docs/specs/xref.yml

```diff
@@ -2162,3 +2162,69 @@ outputs:
   .errors.log: |
     {"message_severity":"warning","code":"missing-attribute","message":"Missing required attribute: 'name'.","file":"docs/TOC.md"}
     {"message_severity":"warning","code":"xref-not-found","message":"Cross reference not found: 'b'.","file":"docs/TOC.md"}
+---
+# files to be redirected must be excluded from xref map
+noSingleFile: true
+repos:
+  https://docs.com/redirected-uid-excluded-from-xrefa#master:
+    - files:
+        docfx.yml:
+        _themes/ContentTemplate/schemas/TestData.schema.json: |
+          {
+            "renderType": "component",
+            "properties": {
+              "uid": { "contentType": "uid" },
+              "fullName": { "type": "string" },
+              "name": { "type": "string" },
+              "title": { "type": "string" }
+            },
+            "xrefProperties": ["title","name","fullName"]
+          }
+        a.yml: |
+          ### YamlMime:TestData
+          uid: a
+          title: Title a.yml
+          name: name a
+          fullName: fullName a
+        b.yml: |
```

Comment:
Consider to have another `c.md`, reference to uid `b` in `c.md`. This uid `b` should not be resolved.